packages/sdk/tasks/finalize-withdrawal.ts

```diff
@@ -0,0 +1,96 @@
+import { task, types } from 'hardhat/config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { Wallet, providers } from 'ethers'
+import { predeploys } from '@eth-optimism/contracts-bedrock'
+import 'hardhat-deploy'
+import '@nomiclabs/hardhat-ethers'
+
+import {
+  CrossChainMessenger,
+  StandardBridgeAdapter,
+  MessageStatus,
+} from '../src'
+
+task('finalize-withdrawal', 'Finalize a withdrawal')
+  .addParam(
+    'transactionHash',
+    'L2 Transaction hash to finalize',
+    '',
+    types.string
+  )
+  .addParam('l2Url', 'L2 HTTP URL', 'http://localhost:9545', types.string)
+  .setAction(async (args, hre: HardhatRuntimeEnvironment) => {
+    const txHash = args.transactionHash
+    if (txHash === '') {
+      console.log('No tx hash')
+    }
+
+    const signers = await hre.ethers.getSigners()
+    if (signers.length === 0) {
+      throw new Error('No configured signers')
+    }
+    const signer = signers[0]
+    const address = await signer.getAddress()
+    console.log(`Using signer: ${address}`)
+
+    const l2Provider = new providers.StaticJsonRpcProvider(args.l2Url)
+    const l2Signer = new Wallet(hre.network.config.accounts[0], l2Provider)
+
+    let Deployment__L1StandardBridgeProxy = await hre.deployments.getOrNull(
+      'L1StandardBridgeProxy'
+    )
+    if (Deployment__L1StandardBridgeProxy === undefined) {
+      Deployment__L1StandardBridgeProxy = await hre.deployments.getOrNull(
+        'Proxy__OVM_L1StandardBridge'
+      )
+    }
```

Comment:
So, if we have both `contracts-bedrock` and `contracts` deployments available, we default to `contracts-bedrock`?
Isn't the main Goerli deployment still on `contracts`, but in the `hardhat.config.ts` file above we've made both available?

Reply:
Both are available so that this script can work with networks that are not legacy